modules/configuration/README.md

```diff
@@ -59,6 +147,8 @@ public static class ChildConfigurationSchema {
   * `rootName` property assigns a _key_ to the root node of the tree that will represent 
     the corresponding configuration schema;
 * `@Config` is similar to the `@ConfigurationRoot` but represents an inner configuration node;
+* `@PolymorphicConfig` is similar to the `@Config` and abstract class in java i.e. cannot be instantiated, but they can be subclassed;
```

Comment:
```suggestion
* `@PolymorphicConfig` is similar to the `@Config` and an abstract class in java, i.e. it cannot be instantiated, but it can be subclassed;
```